scheduler/scheduler_sync.daemon

```diff
@@ -26,7 +26,7 @@
 #       systemctl [command] scheduler_sync@[RADAR_NAME]
 #
 # Dependencies:
-#   - inotifywait (installed by zypper in inotify-tools)
+#   - inotifywait (installed by `zypper in inotify-tools``)
```

Comment:
Just figured I would double check if there should be 2 `` at the end of the line here.